packs/chef/actions/lib/shellhelpers.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import sys
+import os
+import subprocess
+import select
+import io
+
+# Starts command in subprocess processing output as it appears.
+def shell_out(command, env={}):
```

Comment:
Does this look sane now https://github.com/dennybaa/st2contrib/blob/chef-pack/packs/chef/actions/lib/shellhelpers.py ?